vcpkg.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "poisoner",
-  "version-string": "1.0.0",
-  "description": "Simple SKSE mod to add/get poison from weapons. For use with Jayserpa's Smart NPC Potions",
-  "homepage": "https://github.com/JonathanTaylor555/Poisoner",
-  "license": "MIT",
+  "name": "skyrimvresl",
+  "version-string": "0.0.1",
```

Comment:
Suggestion: We should make this 1.0.0, or at least make this 1.0.0 when we release

Reply:
We should go to 1.0.0 when we officially release. I'm thinking about adding semantic_release to this.